docs/layout.rst

```diff
@@ -4,11 +4,9 @@ Advanced dense layouts
 ======================
 
 Tensors (:ref:`scalar_tensor`) can be *placed* in a specific shape and *layout*.
-Defining a proper layout can be critical to performance, especially for memory-bound applications, although in most cases, you probably don't have to worry about it.
-A carefully designed data layout can significantly improve cache/TLB-hit rates and cacheline utilization.
+Defining a proper layout can be critical to performance, especially for memory-bound applications. A carefully designed data layout can significantly improve cache/TLB-hit rates and cacheline utilization. Although in most cases, you probably don't have to worry about it.
```

Comment:
```suggestion
Defining a proper layout can be critical to performance, especially for memory-bound applications. A carefully designed data layout can significantly improve cache/TLB-hit rates and cacheline utilization. Of course, if you don't care about performance, then you probably don't have to worry about advanced data layouts.
```

Reply:
Let's use `most cases`, since **performance is not all about data layouts**, also feel this a bit impolite to those newcomers with adv_layout, wdyt?

Reply:
`most cases` sounds a little vague to me. How about `When performance is not the first priority`?